internal/namespaces/object/v1/s3configfile.go

```diff
@@ -87,16 +88,24 @@ func newS3Config(ctx context.Context, region scw.Region, name string) (s3config,
 	if !accessExists {
 		return s3config{}, errors.New("no access key found")
 	}
+
 	secretKey, secretExists := client.GetSecretKey()
 	if !secretExists {
 		return s3config{}, errors.New("no secret key found")
 	}
+
+	projectID, exists := client.GetDefaultProjectID()
+	if !exists {
+		return s3config{}, errors.New("no project ID found")
+	}
+
 	config := s3config{
 		AccessKey: accessKey,
 		SecretKey: secretKey,
 		Region:    region,
 		Name:      name,
 		ctx:       ctx,
+		ProjectID: projectID,
```

Comment:
suggestion: optionally append project ID to the access key

```suggestion
        if projectID != "" {
                accessKey+="@"+projectID
        }
	config := s3config{
		AccessKey: accessKey,
		SecretKey: secretKey,
		Region:    region,
		Name:      name,
		ctx:       ctx,
```